<launch>
    <param name="r" type="double" value="0.07" />
    <param name="l" type="double" value="0.2" />
    <param name="w" type="double" value="0.169" />
    <param name="T" type="double" value="5" />
    <param name="N" type="int" value="42" />

    <node pkg="project1" type="odometry_pub" output="screen" name="my_publisher" />
    <node pkg="rosbag" type="play" name="player" args="$(find project1)/bags/bag1.bag"/>

</launch>
